internal/operands/template-validator/resources.go

```diff
@@ -34,6 +36,12 @@ func commonLabels() map[string]string {
 	}
 }
 
+func podLabels() map[string]string {
+	labels := commonLabels()
+	labels[PrometheusLabel] = ""
```

Comment:
do you want the label empty or gone?
If gone you could do `delete(labels, PrometheusLabel)` instead.

Reply:
Nvm, I think I misunderstood what you are trying to achieve here.

Reply:
Honestly this part is still confusing.
Why are you setting this label to empty?

Reply:
Because this how kubevirt defined the Prometheus server to pick it up.

https://github.com/kubevirt/kubevirt/blob/099be903803556bfe3a85075a7b55f0a711d9ca7/pkg/virt-operator/resource/generate/components/crds.go#L506

https://github.com/kubevirt/kubevirt/blob/099be903803556bfe3a85075a7b55f0a711d9ca7/pkg/virt-operator/resource/generate/components/crds.go#L43

and it works :)

Reply:
Alright, thanks for clarifying. Fascinating!